### PR TITLE
[Feat]:Home Page added

### PR DESCRIPTION
**Pull request by t007rushi**
Started E-commerce Project : 

✅Added Starting page With action buttons to explore categories
✅Header added to navigate 
✅Images to showcase Explore more
✅and Footer with contact details & social connect

### CODE REVIEW

**Review by omgaikwad — COMMENTED** [review]
The code looks good, UI is also good.

File: components/homepg.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+.home-descr h1 {
+  color: var(--hmpg-secondary-color);
+}
+
+.home-descr p {
+  font-size: 1.5rem;
+  color: var(--offline-color);
+}
+
+.bold {
```
Comment (horsemaker) [inline components/homepg.css:63]: seems like a generic class name, you could add this in your component lib too

**Review by horsemaker — APPROVED** [review]

File: components/homepg.css (hunk 1 above)
Comment (t007rushi) [reply components/homepg.css:63]: Yes,I must. refactor needed in utility classes

**Review by narharikale — APPROVED** [review]
UI looks smooth 🤩

**Pull request merged**